test/unit/lodash.spec.js

```diff
@@ -27,7 +25,7 @@ describe('Lodash training Object', function ()
             expect(typeof elem1).to.eql('object');
             expect(elem2 instanceof Object).to.eql(true);
         });
-        it('should check if first object has right properties', function(){
+        it('should check if object has right properties', function(){
```

Comment:
why U removed 'first'?

It was better that way, in this tests You could use first object, second object or elem1, elem2 (but first would be better, because from command line where You run tests You cannot see how You are calling variables so it would be more readable)